recipes/umami/meta.yaml

```
@@ -0,0 +1,59 @@
+
+{% set version = "0.1" %}
+
+package:
+  name: umami
+  version: {{ version }}
+
+source:
+  url: https://github.com/TerrainBento/umami/archive/v{{ version }}.tar.gz
+  sha256: 2d4a12da9f52df749511eb349d9033e9dac3596bb0792dbed2aeb49be53b917a
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python>=3.6
```

Please add a space between package name and version spec (please do this for all requirements)
```suggestion
    - python >=3.6
```

---

Done